contrib/vagrant/scripts/helpers.bash

```diff
@@ -56,10 +56,10 @@ kubernetes_master="${controllers_ips[0]}"
 # Default values for IPv4 and IPv6
 #
 # CIDR Range for Pods in cluster.
-k8s_cluster_cidr=${K8S_CLUSTER_CIDR:-"10.16.0.0/12,FD02::/96"} # 10.16.0.1-10.31.255.255
+k8s_cluster_cidr=${K8S_CLUSTER_CIDR:-"10.11.0.0/24,FD04::/112"} # 10.11.0.1-10.11.0.255
 # Mask size for node cidr in cluster.
-k8s_node_cidr_v4_mask_size=${K8S_NODE_CIDR_MASK_SIZE:-"16"} # 1st Node: 10.16.0.1-10.16.255.254, 2nd Node: 10.17.0.1-10.17.255.254...
-k8s_node_cidr_v6_mask_size=${K8S_NODE_CIDR_V6_MASK_SIZE:-"112"} # 1st Node: 10.16.0.1-10.16.255.254, 2nd Node: 10.17.0.1-10.17.255.254...
+k8s_node_cidr_v4_mask_size=${K8S_NODE_CIDR_MASK_SIZE:-"24"} # 1st Node: 10.11.0.1-10.11.0.255, 2nd Node: 10.11.1.1-10.11.1.255...
+k8s_node_cidr_v6_mask_size=${K8S_NODE_CIDR_V6_MASK_SIZE:-"112"} # 1st Node: 10.11.0.1-10.11.0.255, 2nd Node: 10.11.1.1-10.11.1.255...
```

Comment:
This doesn't seem correct - now each per-node Pod CIDR would have the same size as the cluster's total Pod CIDR.

Did the CIDR sizes actually need fixing?

Reply:
> This doesn't seem correct - now each per-node Pod CIDR would have the same size as the cluster's total Pod CIDR.
> 
> Did the CIDR sizes actually need fixing?

Thanks for the review.
I have reduced the mask value "K8S_CLUSTER_CIDR" by 4 bits